examples/xgboost/iris-classifier/xgboost.ipynb

```diff
@@ -234,7 +234,7 @@
       },
       "source": [
         "<!-- CORTEX_VERSION_MINOR -->\n",
-        "That's it! See the [example on GitHub](https://github.com/cortexlabs/cortex/tree/master/examples/iris-classifier) for how to deploy the model as an API."
+        "That's it! See the [example](https://www.cortex.dev/iris-classifier) for how to deploy the model as an API."
```

Comment:
I think this should be `https://github.com/cortexlabs/cortex/tree/master/examples/xgboost/iris-classifier`